packages/playground/public/info/contracts_list.md

```diff
@@ -2,12 +2,13 @@
 title: Contracts
 ---
 
-You can find detailed information about your [node/gateway contracts](https://manual.grid.tf/tfchain/tfchain.html)
+For more details about Contract Types, Billing Cycle & Grace Period, check [Contract Documentation](https://manual.grid.tf/tfchain/tfchain.html)
+<br />
+
+To explore further contract details, check [Node Contract Documentation](https://manual.grid.tf/playground/contracts_list.html)
+<br />
 
 - Contract State: Created, GracePeriod (and also know how many tokens are needed to fund contracts in graceperiod to resume your workloads), and the expiration date of these contracts
```

Comment:
"Contract State provides you with the following information:
   - an indicator of whether the contract is created or in the grace period
   - the number of tokens needed to fund contracts in the grace period to resume the workload
   - contract expiration date"